pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchClient.java

```diff
@@ -96,6 +110,12 @@ public void afterBulk(long executionId, List<BulkProcessor.BulkOperationRequest>
         this.client = retry(() -> RestClientFactory.createClient(config, bulkListener), -1, "client creation");
     }
 
+    public void incrementCounter(String counter, double value) {
```

Comment:
why public?

Reply:
Because it will be called by ElasticSearchSink.java too

Reply:
Move the function to the metrics class as you suggested